whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/NrtmVersionInfoRepository.java

```diff
@@ -0,0 +1,128 @@
+package net.ripe.db.nrtm4.persist;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.sql.PreparedStatement;
+import java.sql.Statement;
+import java.util.Optional;
+import java.util.UUID;
+
+
+@Repository
+public class NrtmVersionInfoRepository {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NrtmVersionInfoRepository.class);
+    private final JdbcTemplate jdbcTemplate;
+    private final RowMapper<VersionInformation> rowMapper = (rs, rowNum) ->
+        new VersionInformation(
+            rs.getLong(1),
```

Comment:
You have duplicated this rowMapper in repository layer as well. May be create a class out of it? And also can we have all insert or select statement in Dao, its easier to maintain at one class